common/script/index.js

```diff
@@ -2281,13 +2281,14 @@ api.wrap = function(user, main) {
           user.stats.lvl++;
           tnl = api.tnl(user.stats.lvl);
           user.stats.hp = 50;
-          if (user.stats.lvl > api.maxLevel) {
+          var totalStatPoints = user.stats.str + user.stats.int + user.stats.con + user.stats.per;
+          if (totalStatPoints >= api.maxLevel) {
```

Comment:
Ohhh. Sorry. I mixed up what you said with the maxLevel because of the comment's location. I set that up right now.